Clarification around native installation and docker (#11367)
PrairieLearn/PrairieLearn: docs/installingNative.md

# Running natively

> This page describes the procedure to install and run PrairieLearn without any use of Docker. This means that PrairieLearn is running fully natively on the local OS. Certain features, such as external graders and workspaces, still require Docker if you intend to use them. PrairieLearn supports native execution on macOS, Linux, and Windows inside [WSL 2](https://learn.microsoft.com/en-us/windows/wsl/install).

Reviewer: ```suggestion
This page describes the procedure to install and run PrairieLearn fully natively without using Docker. Certain features, such as external graders and workspaces, still require Docker. PrairieLearn supports native execution on macOS, Linux, and Windows inside [WSL 2](https://learn.microsoft.com/en-us/windows/wsl/install).
```

  make test
  ```

>   _Note:_ Docker is required to pass the test suite

Reviewer: Nit, move this note up to after "Run the test suite", maybe this:

> Run the test suite (Docker must be installed and running):

Author: https://github.com/PrairieLearn/PrairieLearn/issues/11125

  make start
  ```

> - If you need support for [workspaces](workspaces/index.md), start Docker first, and then in a separate terminal run:

Reviewer: ```suggestion
- If you need support for [workspaces](workspaces/index.md), ensure Docker is installed and running, and then in a separate terminal run:
```
